src/orquestra/integrations/cirq/conversions/_cirq_pauli_conversions.py

```diff
@@ -4,7 +4,10 @@
 import warnings
 from typing import List, Optional, Union
 
-import cirq
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    # Pandas throws deprecation warning related to pyarrow
+    import cirq
```

Comment:
Suggestion added in https://github.com/zapatacomputing/orquestra-cirq/pull/54

Reply:
😭 looks like `flake8p` [doesn't like my approach](https://github.com/zapatacomputing/orquestra-cirq/actions/runs/7783989277/job/21223609268?pr=54)

Reply:
I would be fine with you just asking flake8p to ignore those lines where it is complaining. Silencing those warnings in a safe way is worth it.